gui/resources.xml

```diff
@@ -1289,7 +1319,7 @@ Vocabulary
           <property name="icon-name">mypaint-tool-hsvsquare</property>
           <property name="label" translatable="yes" context="Menu→Color (labels), Accel Editor (labels)">HSV Square</property>
           <property name="tooltip" translatable="yes" context="Accel Editor (descriptions)">Dockable color selector: HSV square which can be rotated to show different hues.</property>
-          <!-- FIXME: this description needs to be improved -->
+          <!-- FIXME: this description needs to be improved Note-ALbert: Looks good to me.-->
```

Comment:
Then how about " Dockable color selector: HSV square that has a ring around which can be rotated to show different hues."?